app/gui2/src/util/file.ts

```diff
@@ -0,0 +1,4 @@
+export function fileName(path: string) {
```

Comment:
What does this function do? Please add documentation.

Reply:
i think this can be removed for now actually, it's supposed to extract the file name from a path but it's only needed if the visualization is missing an `export name`